api/src/paths/project/{projectId}/survey/{surveyId}/critters/spatial.ts

```diff
@@ -0,0 +1,207 @@
+    200: {
+      description: 'Geometry of animal captures and mortalities',
+      content: {
+        'application/json': {
+          schema: {
+            type: 'object',
+            additionalProperties: false,
+            nullable: true,
+            required: ['captures', 'mortalities'],
+            properties: {
+              captures: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  additionalProperties: false,
+                  required: ['capture_id', 'geometry'],
+                  properties: {
+                    capture_id: {
+                      type: 'string'
+                    },
+                    geometry: {
+                      type: 'object'
+                    }
+                  }
+                }
+              },
+              mortalities: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  additionalProperties: false,
+                  required: ['mortality_id', 'geometry'],
+                  properties: {
+                    mortality_id: {
+                      type: 'string'
+                    },
+                    geometry: {
+                      type: 'object'
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    400: {
+      $ref: '#/components/responses/400'
+    },
+    401: {
+      $ref: '#/components/responses/401'
+    },
+    403: {
+      $ref: '#/components/responses/403'
+    },
+    500: {
+      $ref: '#/components/responses/500'
+    },
+    default: {
+      $ref: '#/components/responses/default'
+    }
+  }
+};
+
+/**
+ * Fetch geometry of all captures and mortalities for critters in Survey
+ *
+ * @export
+ * @return {*}  {RequestHandler}
+ */
+export function getCritterGeometry(): RequestHandler {
+  return async (req, res) => {
+    const surveyId = Number(req.params.surveyId);
+
+    defaultLog.debug({ label: 'getCritterGeometry', surveyId });
+
+    const connection = getDBConnection(req['keycloak_token']);
+
+    try {
+      await connection.open();
+
+      const surveyService = new SurveyCritterService(connection);
+
+      const surveyCritters = await surveyService.getCrittersInSurvey(surveyId);
+
+      const critterbaseService = new CritterbaseService({
+        keycloak_guid: connection.systemUserGUID(),
+        username: connection.systemUserIdentifier()
+      });
+
+      const critters = await critterbaseService.getMultipleCrittersByIdsDetailed(
+        surveyCritters.map((critter) => critter.critterbase_critter_id)
+      );
+
+      // Exit early if there are no critters
+      if (critters.length < 1) {
+        return res.status(200).json({ captures: [], mortalities: [] });
+      }
+
+      const captures: ICaptureGeometry[] = [];
+
+      // Map over all captures to extract geometry as points
+      for (const critter of critters) {
```

Comment:
Should this logic be moved to its own service method?

Reply:
It looks like this endpoint isn't ever called. So might be able to remove it entirely.

Reply:
In the meantime, I've put a TODO on the endpoint to flag that its unused, and can maybe be removed.

Reply:
I created `/critters/spatial` in SIMS before realizing it should be Critterbase with `/critters/spatial`, so yes, no problem removing it